app.js

```diff
@@ -80,7 +80,8 @@ app.post('/api/message', function (req, res) {
   // Send the input to the assistant service
   assistant.message(payload, function (err, data) {
     if (err) {
-      return res.status(err.code || 500).json(err);
+      var status = (err.code  !== undefined && err.code > 0)? err.code : 500;
```

Comment:
It's the same comment as https://github.com/watson-developer-cloud/assistant-simple/pull/233#pullrequestreview-227758865

Reply:
fixed